docs_src/basic_train.ipynb

```diff
@@ -2863,6 +2863,13 @@
     "show_doc(Recorder.plot_lr)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The learning rate and momentum shown are that of the last layer (```opt.lr``` and ```opt.mom```)."
```

Comment:
```suggestion
    "The learning rate and momentum shown are that of the last layer (`opt.lr` and `opt.mom`)."
```

Reply:
Note that single back quotes are the use in the documentation notebooks.

Reply:
Got it. Will keep this in mind for future PRs. Thanks.